psql2mysql/__init__.py

```diff
@@ -289,6 +322,15 @@ def add_subcommands(subparsers):
     parser = subparsers.add_parser(
         'migrate',
         help='Migrate data from PostgreSQL to MariaDB')
+
+    parser.add_argument("--chunked",
```

Comment:
Shouldn't this be added directly into`cli_opts`

Reply:
I added it here as these options only relate to the migrate sub command, if I add it at `cli_opts` they'll appear as global options.

Reply:
OK, I guess you are right